src/app.ts

```diff
@@ -605,6 +605,12 @@ app.post(
   safeRequestHandler(standingOrdersAPI.triggerStandingOrderRequestHandler)
 );
 
+// BACKOFFICE - SEIZURES PROTECTION
+app.post(
+  "/__BACKOFFICE__/setAccountIsPkonto/:personId",
```

Comment:
Meeting note: this sets "seizure_protection" data to some sample object, so we can rename